README.md

```diff
@@ -38,17 +40,16 @@ Adjust your activity to inherit from `FlutterFragmentActivity`:
       // ...
   }
   ```
-#### For card Component only:
-
-Declare the intent filter in your `AndroidManifest.xml` file:
+#### For Components:
+Declare the intent filter in your `AndroidManifest.xml` file for every component you are using:
   ```xml
     <intent-filter>
         <action android:name="android.intent.action.VIEW" />
         <category android:name="android.intent.category.DEFAULT" />
         <category android:name="android.intent.category.BROWSABLE" />
         <data
             android:host="YOUR_APPLICATION_ID e.g. com.adyen.checkout.flutter.example"
-            android:path="YOUR_CUSTOM_PATH e.g. /card"
+            android:path="YOUR_CUSTOM_PATH e.g. /card or /googlePay"
             android:scheme="adyencheckout" />
```

Comment:
This should probably be different for session and non-session flows 🤔
Otherwise on `session` is super limiting for components..

Reply:
Let's discuss this and go through the code to see if this is really the case.